### PR TITLE
Use caret versions in package.json

### PR DESCRIPTION
**Pull request by TikiTDO**
Closes #1231

### CODE REVIEW

**Comment by TikiTDO** [comment]
There's also a couple of outdated packages even after the update. Looks like the git hooks integrator moved up a version, and some typescript type got bumped:

```
└> npm outdated
Package        Current  Wanted  Latest  Location                    Depended by
@types/globby    8.0.0   8.0.0   9.1.0  node_modules/@types/globby  sortier
husky            5.1.3   5.2.0   6.0.0  node_modules/husky          sortier
```

Neither seem super critical, but figured I'd mention it here.

File: package.json [hunk 1]
```diff
@@ -4,41 +4,41 @@
...
-    "ignore": "5.1.8",
-    "postcss-less": "4.0.1",
-    "postcss-scss": "3.0.5",
-    "typescript": "4.2.4"
+    "@types/postcss-less": "^4.0.0",
+    "@typescript-eslint/typescript-estree": "^4.24.0",
+    "angular-html-parser": "^1.8.0",
+    "cosmiconfig": "^7.0.0",
+    "find-up": "^5.0.0",
+    "flow-parser": "^0.151.0",
```
Comment (k2snowman69) [inline package.json:12]: Beta packages are a bit less stable
```suggestion
    "flow-parser": "0.151.0",
```

File: package.json [hunk 2]
```diff
@@ -4,41 +4,41 @@
...
-    "husky": "5.1.3",
-    "jest": "26.6.3",
-    "lint-staged": "11.0.0",
-    "prettier": "2.3.0",
-    "rimraf": "3.0.2",
-    "source-map-support": "0.5.19",
-    "ts-jest": "26.5.6",
-    "ts-node": "9.1.1",
-    "tslint": "6.1.3"
+    "@snowcoders/renovate-config": "^2.0.3",
```
Comment (k2snowman69) [inline package.json:21]: Pin all dev dependencies, these aren't shipped so the less unstable the better. See https://docs.renovatebot.com/dependency-pinning/#so-whats-best for details of when to pin and when not to pin

**Review by k2snowman69 — CHANGES_REQUESTED** [review]

**Comment by k2snowman69** [comment]
Just to speed things up and unblock you I've opened https://github.com/snowcoders/sortier/pull/1236

**Comment by k2snowman69** [comment]
Just published 3.3.0, let me know how it goes for you

**Comment by TikiTDO** [comment]
Thanks, I'm out for the long weekend here, but I'll check it out on Tuesday

**Comment by TikiTDO** [comment]
New version works like a charm, many thanks

**Pull request closed without merging**